modules/transactionpool/persist.go

```diff
@@ -53,6 +55,9 @@ func (tp *TransactionPool) initPersist() error {
 	}
 
 	// Open the database file.
+	if build.DEBUG && build.Release != "testing" {
+		fmt.Println("\tTransaction Pool: opening database")
+	}
```

Comment:
we could set it up so that the profiler is running in DEBUG mode, I'm not sure how expensive that would be, but should help.